### ISSUE TITLE
brainstorming TOS

### ISSUE THREAD

**Issue by ainigrivS** [OpenCollective/OpenCollective#71]
Dispute Mechanism 
Do we need a dispute mechanism? 
What’s our policy when there is conflict between hosts and collectives? 
To what extent are we going to get involved? 

To Read: Kickstarter ToS

Hosts Manual 
What can we advise hosts to be careful with?
Do we advise them to get insurance? 

To Read: Airbnb ToS for hosts 

Virginia’s comments & notes:

From Heroku:
https://www.heroku.com/policy/tos
https://www.heroku.com/policy/aup
https://www.heroku.com/policy/privacy

Define OC services – service available through OC’s website and any services provided in connection to any of those services

Agreement to the ToS by actually using the services.

Who cannot use the services? People over 18. People who are barred from receiving the services by the laws of their country or the US or any other laws that might otherwise be applicable.

Exclusion from provision of services: eg certain types of associations (hate based, illegal, etc.)

From Stripe: 
Stripe has great streamlined and in plain English : 
https://stripe.com/us/terms
https://stripe.com/us/privacy

question: do you retain the transaction history? Or do you request the customer to retain the transaction history? If OC retains it – check data retention policy & compliance with safe harbor regulations (US-EU)

because the use of OC services are reliant on Stripe for processing of payments, the user must agree to comply with all the terms of services of Stripe. 

IP – side:
License to use OC services & trademarks

Kickstarter terms:
https://www.kickstarter.com/trust?ref=footer
their format is great because they explain in plain English what a section is about and then have the lagelese: https://www.kickstarter.com/terms-of-use?ref=footer 

E&Y Privacy policy US:
http://www.ey.com/US/en/home/privacy 
http://www.ey.com/Publication/vwLUAssetsPI/EY_Data_Protection_BCR_Programme/$FILE/EY-Data-Protection-BCR-Programme.pdf

Safe harbor is invalid, but the department of commerce continues to administer the SH program: http://www.export.gov/safeharbor/ 

PWC privacy: http://www.pwc.com/us/en/site/privacy.html 

Amazon cloud:
https://aws.amazon.com/legal/?nc1=f_cc
Include disclaimer that a Host is NOT an agent https://aws.amazon.com/agreement/ 
https://aws.amazon.com/service-terms/ 

Compliance with regulations: 
KYC
Antiboycott laws
Export regulations (export of data and saas)
ITAR(maybe)
http://export.gov/eac/show_detail_trade_events.asp?EventID=35037&InputType=EVENT 

Example of SAAS ToS(legalese):
https://saas.agi.com/analyticalServices/AGI-terms-conditions.html 

Host agreement: 

Closing of a collective – what happens with the funds: 

In the subscription, inform the collective that in case they shut down, the remaining money less the fees will be donated to another collective assigned by the contributors or, if such collective is not live, then the money will be donated to another collective with a similar mission. 

Conflict resolution. 
We are facilitators not mediators or supervisors. In order to focus on our mission, we cannot allocate resources to dispute resolution between users. 

Our Service Providers 

In order to provide you with our services we work with several other service providers. They require compliance with their own terms of service. By using our services, you agree to comply with the terms of our providers. Please check them here:

Stripe https://stripe.com/us/terms 

Paypal …
Heroku
Mailchimp… 

Agency
You are responsible for your own decisions regarding the management and spending of the funds. We are not acting as your agent or your principal. 

Regulatory compliance:

All users must comply with all US and non-US regulations applicable to them, 

In legalese, specify:
>>> including the following:

-	anti boycott laws and OAC regulations (may be very important in the context of your business), 
-	 export control regulations, 
-	international traffic of arms regulations(maybe.. but not sure yet), 
-	regulations of the office of foreign asset control. 

<<<

Governing law
We are operating from NY, so all the disputes will be handled under NY law in NY. 

Prohibited items list – PIA. 

Legalese – Virginia. 

Timeline - end of May.

**Comment by piamancini** [comment]
new draft already, being reviewed.
